docs/en/corne-cherry-v3.md

```diff
@@ -1 +1,126 @@
-Sorry!! WIP!! In 3-5 days, will be published in English version.
+# Preparation for Corne Cherry v3 PCB
+
+## Items Needed for PCB Assembly
+
+| Name                                              | Quantity           |
+| ------------------------------------------------- | ------------------ |
+| Corne Cherry v3 PCB                               | 1 set (left and right) |
+| Surface mount diodes (1N4148W)                    | 42                 |
+| PCB sockets for MX                                | 42                 |
+| TRRS jack (PJ-320A)                               | 2                  |
+| OLED 128x32                                       | 2                  |
+| Compatible MCU                                    | 2                  |
+| MCU conthrough(spring pin headers) or pin socket (height 2.5mm)             | 4                  |
+| Tactile switches for reset (2 pin)                | 2                  |
+
+---
+
+The following items are necessary but included with Corchim:
+
+| Name                   | Quantity |
+| ---------------------- | -------- |
+| Pin sockets for OLED   | 2        |
+| Pin headers for OLED   | 2        |
+
+### A Note About the Color of Kailh MX Sockets
+
+Since I was asked about this, the color doesn't matter; it's just fashion (although you might not see it with the case on). Places to purchase include:
+
+- [Daily Craft Shop](https://shop.dailycraft.jp/products/keyswitch_socket_transparent)
+- [Chosfox](https://chosfox.com/products/kailh-colorful-hot-swap-socket)
+- [AliExpress](https://ja.aliexpress.com/item/1005003174680450.html)
+
+### Compatible MCUs
+
+- Elite-C
+- Elite-Pi
+- key micro
+- Sea-Picro
+
+For the key micro and Sea-Picro, we have confirmed that they operate without soldering on the MCU side. However, using Sea-Picro without soldering is not recommended due to its tendency to detach easily.
+
+### About MCU conthrough
+
+Due to the USB-C port, we recommend a height of 2.5mm.
+
+## Assembly Steps
+
+### 1. Soldering Diodes and Sockets
+
+![Soldering Diodes and Sockets](../img/corne-cherry-v3/pcb-diode-socket.jpg)
+
+The soldering of [diodes](https://github.com/foostan/crkbd/blob/main/corne-cherry/doc/v3/buildguide_jp.md#%E3%83%80%E3%82%A4%E3%82%AA%E3%83%BC%E3%83%89) and [sockets](https://github.com/foostan/crkbd/blob/main/corne-cherry/doc/v3/buildguide_jp.md#pcb%E3%82%BD%E3%82%B1%E3%83%83%E3%83%88) follows the same steps as the [original Corne](https://github.com/foostan/crkbd/blob/main/corne-cherry/doc/v3/buildguide_jp.md). LEDs are not supported, but having them is not a problem (more on this later).
```

Comment:
If possible, pls replace the links to original corne docs in English.

- https://github.com/foostan/crkbd/blob/main/corne-cherry/doc/v3/buildguide_en.md#diodes
- https://github.com/foostan/crkbd/blob/main/corne-cherry/doc/v3/buildguide_en.md#switch-sockets
- https://github.com/foostan/crkbd/blob/main/corne-cherry/doc/v3/buildguide_en.md

Reply:
fixed!
thanks review!